playbooks/roles/os_temps/tasks/main.yml

```diff
@@ -13,7 +13,7 @@
 
 # Get ip of the cluster unless it was provided
 - name: "Get the cluster Server URL for project {{ openshift_project }}"
-  shell: "{{ oc_bin }} status | egrep '{{ openshift_project }}.*server' | awk '{print $NF}'"
+  shell: "{{ oc_bin }} status | grep 'https' | awk '{print $NF}'"
```

Comment:
probably should have a ` | head -1` after the `awk`

Reply:
Thanks and done